services/frontend/src/i18n/en_CA.json

```diff
@@ -343,6 +343,14 @@
   "search.advanced.general.title": "General info",
   "search.advanced.skill.title": "Skills and talent",
   "search.empty.query": "Refine your search criterias to start finding talent!",
+  "search.filter.name": "Input Name",
```

Comment:
I would rename this to `Person name` or something, to be more descriptive, it would be `Nom de personne` in French

Reply:
Fixed.